test/unit/math/rev/mat/fun/matrix_exp_action_handler_test.cpp

```diff
@@ -0,0 +1,128 @@
+#include <stan/math/rev/mat.hpp>
```

Comment:
This is only the `double` value tests.  I think it can be moved along with the implementation.

Reply:
removed.